Common/packages.config

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0" targetFramework="net452" />
-  <package id="Microsoft.CodeAnalysis.Common" version="1.0.0" targetFramework="net452" />
-  <package id="Microsoft.CodeAnalysis.CSharp" version="1.0.0" targetFramework="net452" />
-  <package id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="1.0.0" targetFramework="net452" />
-  <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="1.0.0" targetFramework="net452" />
-  <package id="Microsoft.Composition" version="1.0.27" targetFramework="net452" />
-  <package id="System.Collections.Immutable" version="1.1.36" targetFramework="net452" />
-  <package id="System.Reflection.Metadata" version="1.0.21" targetFramework="net452" />
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0" targetFramework="net46" />
```

Comment:
Changing the target project framework through the VS UI doesn't update the package config automatically, so I manually edited the package config files.